fastlane_core/lib/fastlane_core/command_executor.rb

```diff
@@ -15,11 +15,12 @@ def which(cmd)
         # PATHEXT contains the list of file extensions that Windows considers executable, semicolon separated.
         # e.g. ".COM;.EXE;.BAT;.CMD"
         exts = ENV['PATHEXT'] ? ENV['PATHEXT'].split(';') : ['']
+        exts << ''
```

Comment:
what does this do?

Reply:
It adds "no file extension" to the list. This was needed for a specific test... that I just now can't remember or pinpoint. Will try to find out again.

Reply:
Looking at the code again I think I just wrote that in a suoptimal way.
`exts` should always contain empty string as a value, but get the result of the `.split` on windows additionally. 
How would I write this in way that is more obvious and not confusing?

Reply:
1 it is - that is 100% clear and readable. Thanks!